### PR TITLE
Upgrades KDS and dependencies

### PR DESCRIPTION
**Pull request by indirectlylit**
## Summary

* Upgrade vue from 2.6.11 -> 2.6.14 in line with KDS
* Remove counterproductive `kolibri-tools` references which were pinning to old versions rather than using yarn workspace symlinks
* Allow browserlist-config-kolibri to be published
* Upgrade private packages to 0.15.0-dev.4 (see follow-up issue #8454)
* Upgrade KDS to v1.0.0

## References

* https://github.com/learningequality/kolibri-design-system/releases/tag/v1.0.0
* supersedes #8085

## Reviewer guidance

Sanity-check to make sure things run as well as they did prior to this commit

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [x] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: packages/kolibri-tools/package.json [hunk 1]
```diff
@@ -101,9 +101,6 @@
     "node": "10.x",
     "npm": ">= 3"
   },
-  "optionalDependencies": {
-    "kolibri": "0.13.0-dev.10"
```
Comment (rtibbles) [inline packages/kolibri-tools/package.json:105]: I think given that we want to get rid of this package, this is probably OK - but just being aware that outside of the `packages` workspace context, I can imagine this being an issue, as there's no indication that this should be installed.

**Review by rtibbles — CHANGES_REQUESTED** [review]
Should clean up the code that does this version setting logic as well as the versions themselves:
https://github.com/learningequality/kolibri/blob/release-v0.15.x/packages/publish.js#L57

Aside from this, the asset has built fine and runs as expected.

File: packages/kolibri-tools/package.json (hunk 1 above)
Comment (indirectlylit) [reply packages/kolibri-tools/package.json:105]: why is it optional?

Comment (rtibbles) [reply packages/kolibri-tools/package.json:105]: Because it's only needed for running tests, not for linting or building plugins.

**Comment by indirectlylit** [comment]
updated with cleanup to _publish.js_

**Review by rtibbles — APPROVED** [review]

**Pull request merged**